common/src/Model/Node.h

```diff
@@ -99,6 +99,8 @@ namespace TrenchBroom {
             const NodeList& children() const;
             size_t descendantCount() const;
             size_t familySize() const;
+
+            bool addToNodeTree() const;
```

Comment:
This sounds like a command. `shouldAddToNodeTree()` instead? 

The term `NodeTree` sounds like the parent/child relationship of Node objects, I think `SpatialIndex` would be clearer.

Reply:
Yes, agreed.